src/test/java/dk/kb/datahandler/oai/OaiHarvestClientIntegrationTest.java

```diff
@@ -59,4 +83,42 @@ void testHarvest1000OaiRecords() throws Exception {
         assertEquals(1000, r2.getRecords().size());
     }
 
+
+    @Tag("integration")
+    @Test
+    void testPreservicaSevenAuth() throws Exception {
+
+        YAML conf = ServiceConfig.getConfig();
+
+        // name: pvica6.devel
+        OaiTargetDto oaiTarget = new OaiTargetDto();
+        oaiTarget.setUrl(conf.getString("integration.oaiTargets[1].url")); //Public KB service
+        oaiTarget.setMetadataprefix(conf.getString("integration.oaiTargets[1].metadataPrefix"));
+        oaiTarget.setUsername(conf.getString("integration.oaiTargets[1].user"));
+        oaiTarget.setPassword(conf.getString("integration.oaiTargets[1].password"));;
+        oaiTarget.setDatasource(conf.getString("integration.oaiTargets[1].datasource"));
```

Comment:
Could this be written with a conditional yPath instead? `[name=pvica7.devel]`?

Reply:
Yeah... Looks like an issue for `kb-util`. I have created https://github.com/kb-dk/kb-util/issues/67